checkstyle/suppressions.xml

```diff
@@ -272,6 +272,9 @@
     <suppress checks="MethodCount|NPathComplexity|MagicNumber" files="com/hazelcast/spi/Operation"/>
     <suppress checks="MethodCount" files="com/hazelcast/spi/impl/NodeEngineImpl"/>
     <suppress checks="ClassFanOutComplexity" files="com/hazelcast/spi/impl/NodeEngineImpl"/>
+    <!-- node engine constructs objects, so it knows many of them -->
+    <suppress checks="ClassDataAbstractionCoupling" files="com/hazelcast/spi/impl/NodeEngineImpl"/>
```

Comment:
Another case in point: I already had a PR prepared which removes the blanket XML suppression from the complete `c.h.elastic` hierarchy. The PR was delayed until after the release; but in the meantime another PR was accepted which turned out to add new code to the same package hierarchy, even though in a completely separate module. After rebasing I was welcomed with 300 new CheckStyle errors to take care of.